ogre_mesh_viewer.py

```diff
@@ -209,8 +212,11 @@ def preRenderTargetUpdate(self, evt):
         if ImGui.BeginMainMenuBar():
 
             if ImGui.BeginMenu("File"):
-                if ImGui.MenuItem("Renderer Settings"):
-                    self.show_render_settings = True
+                if ImGui.MenuItem("Open File", "O"):
+                    self.app._open_file_dialog()
+                if ImGui.MenuItem("Select Renderer"):
+                    self.app.getRoot().queueEndRendering()
```

Comment:
debug leftover?

Reply:
I don't know how that got there.

The "Select Renderer" should not be there